python/cuspatial/cuspatial/core/interpolate.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import numpy as np
+
+from cudf import DataFrame, Series
+from cudf.core.index import RangeIndex
+
+from cuspatial._lib.interpolate import (
+    cubicspline_coefficients,
+    cubicspline_interpolate,
+)
+
+
+def _cubic_spline_coefficients(x, y, ids, prefix_sums):
+    x_c = x._column
+    y_c = y._column
+    ids_c = ids._column
+    prefix_c = prefix_sums._column
+    result_table = cubicspline_coefficients(x_c, y_c, ids_c, prefix_c)
+    result_table._index = RangeIndex(result_table._num_rows)
```

Comment:
This feels kinda nasty to reach inside and set the internal attribute of a `Table` here.

Reply:
When this was written a week ago, the `Table` constructor code didn't correctly create an index (a bug) and this was my workaround. I'm not sure if it has been fixed yet I'll have a look at alternatives.

Reply:
Seems to be no-longer-necessary, thank you.